docs/jwst/tweakreg/README.rst

```diff
@@ -101,14 +101,14 @@ The ``tweakreg`` step has the following optional arguments:
   for matching. (Default=15)
 
 * ``searchrad``: A `float` indicating the search radius in arcsec for a match.
-  (Default=1.0)
+  (Default=2.0)
 
 * ``use2dhist``: A boolean indicating whether to use 2D histogram to find
   initial offset. (Default=True)
 
-* ``separation``: Minimum object separation in arcsec. (Default=0.5)
+* ``separation``: Minimum object separation in arcsec. (Default=1.0)
```

Comment:
For NIRCam short-wave detectors, with a pixel scale of ~0.03", this implies a separation of ~33 pixels. So this setting means that detected sources that are closer to one another than 33 pixels will not be used in the solution? Given that you have 2048x2048 pixel space to work with, I guess this may be reasonable and still give you plenty of sources to work with, but it seems to me (IMHO) that you could have very valid results for sources closer together than that, especially since the NIRCam short-wave detectors are undersampled at short wavelengths (hence the PSF is very compact). But I'll leave it to folks with more experience at this to decide.

Reply:
I guess this is also a strong motivation for having parameter ref files in CRDS that tailor the defaults to each detector/filter/wavelength combination.

Reply:
See my comment below - https://github.com/spacetelescope/jwst/pull/6809#issuecomment-1112252807 but there is very little difference between instruments in terms of alignment errors in units of arcsec (not pixels). These are mostly dictated by pointing accuracy of the observatory. If you are aligning images to an external catalog such as GAIA - then it is about absolute pointing errors and if you are aligning  a set of images relative to each other - then we are talking about relative error in alignment (which could be smaller then absolute errors) - I should add this to my longer comment below.

Reply:
For alignment, we only need to use sources with the highest probability of being cross-matched correctly as opposed to trying to cross-match every source in the field which can lead to a lot of confusion in crowded fields.   So, a larger separation in concert with reducing the number of sources used for alignment makes sense.